### ISSUE TITLE
[1.20.4] [Plugin Compatibility] TrainCarts + BKCommonLib

### ISSUE THREAD

**Issue by vico93** [MohistMC/Banner#225]
### I have confirmed that ...

- [X] Banner is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Fabric

### Banner version

1.20.4-74

### OS & Java versions

OS: Windows 11 (amd64) version 10.0, Java Version: 17.0.10, Microsoft, Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Microsoft

### Plugins and Mods

```raw
Unable to run the command due to the crash, but basically only Fabric API 0.95.4, [EasyEula 1.0](https://modrinth.com/mod/easy-eula) and Banner build 74 from [Jenkins](https://ci.codemc.io/job/MohistMC/job/Banner-1.20.4/).
```

### Description

Basically, i got curious to test TrainCarts + BKCL in newer builds of Banner for 1.20.4. Unfortunately, seems like either BKCLib or Banner itself arent prepared to support each other.

Crash + latest.log in the link below.

PS.: Pingging @bergerkiller to see if he's interested in track this compatibility issue on his side too.

### Step to reproduce

1. Install the server using Fabric Loader installer;
2. Create `mods` and place Fabric API and EasyEula;
3. Run the server once to generate the basic files;
4. Place Banner on `mods` and try to load the server again;
5. Crash happens 😶 

### Logs

https://gist.github.com/vico93/110f47dfac0fd7adbd639106972450eb

**Comment by vico93** [comment]
Up, is there any updates about this? BKCL still needs stuff to be done on Banner side to load correctly
